docs/SUMMARY.md

```diff
@@ -24,3 +24,123 @@
   - [Read chain state](examples/promise/05_read_storage/README.md)
   - [Make a transfer](examples/promise/07_transfer_dots/README.md)
   - [Display system events](examples/promise/08_system_events/README.md)
+
+<!-- These are hidden, allowing HTML generation but does not appear in the sidebar -->
+
+- [hidden](types/classes/_accountid_.accountid.md)
```

Comment:
This is  beyond horrible, but it is what it is - see the linked issue that related back to "best practices" for these types of hacks. (There are plugins to generate the SUMMARY.md automagically, but we are actually crafting these to only have specific items in the index)